tests/unit/query/js-adapter-test.js

```diff
@@ -687,3 +705,21 @@ test('adapter | js | false predicate | detail predicate', (assert) => {
   assert.ok(result);
   assert.equal(result.length, 0);
 });
+
+test('adapter | js | false predicate | detail predicate', (assert) => {
```

Comment:
Для предыдущего теста можно тоже имя уточнить.
```suggestion
test('adapter | js | false predicate | detail predicate all', (assert) => {
```